From 0a53ee5976bb0ff7f66f6be5cad69e9fe8b92cc8 Mon Sep 17 00:00:00 2001
From: Jia Zhang <zhang.jia@linux.alibaba.com>
Date: Tue, 30 Oct 2018 21:33:22 +0800
Subject: [PATCH 49/55] Ignore *.hash

*.hash should be ignored by git status if ENABLE_SHIM_HASH is
configured.

Signed-off-by: Jia Zhang <zhang.jia@linux.alibaba.com>
---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 312a0e3..dcd3983 100644
--- a/.gitignore
+++ b/.gitignore
@@ -24,6 +24,7 @@ shim_cert.h
 *.srl
 *.srl.old
 *.tar.*
+*.hash
 version.c
 cov-int/
 scan-results/
-- 
2.17.1

